importScripts/sampleRecords/itemData.destination.json

```diff
@@ -11,7 +11,7 @@
       "ca_objects.PhysicalCondition":"Fair \/ Used",
       "ca_objects.Materials":"Oil on pre-grounded Windsor &amp; Newton canvas.",
       "ca_objects.Dimensions.DimensionsValue":"H:44.9cm W:29.6cm",
-      "classificationHierarchy":"Root node for Classification (179); ILLUSTRATIONS (224); Oil (225); portrait (226)"
+      "classificationHierarchy":"Root node for Classification (179); ILLUSTRATIONS (225); Oil (226); portrait (227)"
```

Comment:
These ID's have changed as the `Photograph` classification values are no longer being loaded.